README.md

```diff
@@ -20,17 +16,19 @@ interface using [wysihtml5](https://github.com/xing/wysihtml5).
 ```ruby
 # Gemfile
 
-gem 'activeadmin-dragonfly', github: 'stefanoverna/activeadmin-dragonfly'
+# gem 'activeadmin-dragonfly', github: 'stefanoverna/activeadmin-dragonfly'
```

Comment:
commenting out in the `README.md`? why not just remove the line?

Reply:
We are dropping support to document upload but I feel like we could eventually re-add it with more work.
I kind of wanted to keep that here to find it easily, even if we look at it back in a few months.